apps/android/app/src/main/java/chat/simplex/app/model/SimpleXAPI.kt

```diff
@@ -393,7 +396,7 @@ open class ChatController(private val ctrl: ChatCtrl, private val ntfManager: Nt
         val a = e.agentError
         when {
           a is AgentErrorType.BROKER && a.brokerErr is BrokerErrorType.NETWORK -> "network"
-          a is AgentErrorType.SMP && a.smpErr is SMPErrorType.AUTH -> "contact deleted"
+          a is AgentErrorType.SMP && a.smpErr is SMPErrorType.AUTH -> "contact deleted"  // todo translate?
```

Comment:
yes, both need to be translated as they are shown in the UI (I think?) - check what chatModel.updateNetworkStatus does with it.